README.md

```diff
@@ -126,7 +118,7 @@ Head to the <a href="#live-demo">LIVE DEMO SECTION</a> for Live links and presen
 
 > View the live demo of this project by clicking the text below.
 
-- [See Live Demo](https://geekelo.github.io/awesome-books-webapp/)
+- [See Live Demo](https://geekelo.github.io/oti-task-organizer-webapp/dist/index.html)
 <!-- - [First Presenation Video](https://www.loom.com/share/719c333faa524c5db6930878f33b809b) -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The LICENSE link is broken, kindly make sure to fix it so your README links are working properly